ui/src/app/resources/applications/application-creation.component.ts

```diff
@@ -21,7 +21,8 @@ export class ApplicationCreationComponent {
       () => console.log('application created'),
     )
 
-    console.log(this.things)
+    console.log(this.things);
```

Comment:
This is actually my junk code that seems to have snuck through. >,<

The whole `things` var can be stripped out of here.